### ISSUE TITLE
[SotA Final Scenario Dialog] The protagonist says the previous scenario was 'easy'

### ISSUE THREAD

**Issue by max-torch** [wesnoth/wesnoth#6119]
### Text Issues: Typos, grammar mistakes, suggested improvements

* English Language Text

* SotA Final Scenario

* This [dialog](https://github.com/wesnoth/wesnoth/blob/ca90694141d8e2c5d2d1ea8afcfb071e9a439f12/data/campaigns/Secrets_of_the_Ancients/scenarios/21_Against_the_World.cfg#L632-L639) appears when Dwarves emerge onto the map:
```
        [message]
            speaker=Ras-Tabahn
            message= _ "Look! The dwarves circled around in front of us."
        [/message]
        [message]
            speaker=Ardonna
            message= _ "That’s why getting through Knalga was so easy."
        [/message]
```
The issues with this are:
* The second-to-the-last scenario, North Knalga, is one of the most challenging scenarios in the entire campaign.
* The amount of dwarves that appear in the final scenario is small and looks like a small scout force compared to the massive army of dwarves you fight in the previous scenario.

* _(optional)_ What do you propose to change the text to?
```
        [message]
            speaker=Ardonna
            message= _ "On the bright side, if they did not choose to split their forces and surprise us here we might have faced greater opposition back in Knalga!"
        [/message]
```

### Reference Images
Back in Knalga, first batch of dwarves:
![image](https://user-images.githubusercontent.com/41433185/134893061-33dd1d67-2d25-4d25-a198-ef8d839935c5.png)
Back in Knalga, _second_ batch of dwarves:
![image](https://user-images.githubusercontent.com/41433185/134893888-73eaea23-383b-4157-87b7-546267a59d00.png)

In the Ford of Abez, one small batch of dwarves:
![image](https://user-images.githubusercontent.com/41433185/134893334-ac3f1115-915e-4d5c-bf6e-8f442e4842da.png)

**Comment by Wedge009** [comment]
I haven't played the 1.15 revision to SotA yet, but I wonder if it's meant to be a sarcastic remark or just reflection of what's supposed to be in the story. I often find - in games generally, not specifically Wesnoth - that game-play and story don't always have to line-up.

**Comment by cooljeanius** [comment]
yeah I thought that seemed off when I finally got around to beating SotA recently, too (via cheating, but still)...

cc @beetlenaut

**Comment by knyghtmare** [comment]
Perhaps this word change?
```
        [message]
            speaker=Ardonna
            message= _ "That’s why getting through Knalga was so uneventful."
        [/message]
```

**Comment by max-torch** [comment]
I decided to embrace the spirit of the original text and expound on it. https://github.com/wesnoth/wesnoth/pull/6168/commits/71f141fa11cb7a9e3e5ebc1a32d856a95e3b67e2

**Comment by stevecotton** [comment]
Re-reading the text of S21, they've had about three days walk in Knalga after the end of S20 before they reach the Ford of Abez. It's all covered by the single line "From the northern part of the kingdom of Knalga, the tunnel split into many passageways. Being able to feel the life force of the dwarves allowed us to take mostly unoccupied tunnels, which we did."
